package.json

```diff
@@ -27,28 +27,27 @@
   },
   "private": true,
   "dependencies": {
-    "@angular/animations": "^12.2.16",
-    "@angular/common": "^12.2.16",
-    "@angular/compiler": "^12.2.16",
-    "@angular/core": "^12.2.16",
-    "@angular/platform-browser": "^12.2.16",
-    "@angular/platform-browser-dynamic": "^12.2.16",
-    "@angular/router": "^12.2.16",
-    "@auth0/auth0-spa-js": "^1.21.0",
-    "rxjs": "^6.6.7",
-    "tslib": "^2.4.0",
-    "zone.js": "~0.11.5"
+    "@angular/animations": "^14.0.0",
+    "@angular/common": "^14.0.0",
+    "@angular/compiler": "^14.0.0",
+    "@angular/core": "^14.0.0",
+    "@angular/forms": "^14.0.0",
+    "@angular/platform-browser": "^14.0.0",
+    "@angular/platform-browser-dynamic": "^14.0.0",
+    "@angular/router": "^14.0.0",
+    "@auth0/auth0-spa-js": "^1.22.0",
+    "rxjs": "~6.6.0",
+    "tslib": "^2.3.0",
+    "zone.js": "~0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "^12.0.0",
-    "@angular-devkit/build-ng-packagr": "^0.1000.8",
-    "@angular/cli": "^12.2.13",
-    "@angular/compiler-cli": "^12.2.14",
-    "@angular/forms": "^12.2.14",
+    "@angular-devkit/build-angular": "^14.0.0",
+    "@angular/cli": "^14.0.0",
+    "@angular/compiler-cli": "^14.0.0",
     "@briebug/cypress-schematic": "^4.4.0",
-    "@types/jasmine": "^3.6.2",
+    "@types/jasmine": "~3.8.0",
     "@types/jasminewd2": "~2.0.3",
-    "@types/node": "^12.19.8",
+    "@types/node": "^12.11.1",
```

Comment:
From Angulars changelog for NG 14.

> Support for Node.js v12 has been removed as it will become EOL on 2022-04-30. Please use Node.js v14.15 or later.

Reply:
When creating a new angular 14 project, this dependency does not even get added, I think I will remove it.

Reply:
Let's refrain from removing dependencies if you are not aware of any reason they should or shouldn't be removed.